src/sentry/api/bases/organization.py

```diff
@@ -98,6 +98,13 @@ def convert_args(self, request, organization_slug, *args, **kwargs):
 class OrganizationReleasesBaseEndpoint(OrganizationEndpoint):
     permission_classes = (OrganizationReleasePermission,)
 
+    def has_invalid_repos(self, refs, organization):
+        repos = set(Repository.objects.filter(
+            organization_id=organization.id,
+            name__in=[r['repository'] for r in refs],
+        ).values_list('name', flat=True))
+        return bool([r['repository'] for r in refs if r['repository'] not in repos])
```

Comment:
Nevermind, re-reading what you're actually trying to do here.

Why don't we just compare counts? Like:

```python
names = [r['repository'] for r in refs]
return Repository.objects.filter(
    organization_id=organization.id,
    name__in=names,
).count() == len(names)
```

This would be a bit quicker and we are just checking if they were all found.